Model/Transaction.php

```diff
@@ -274,14 +274,18 @@ protected function buildLineItems($order, $items, $type = 'order') {
                 'sales_tax' => $tax
             ];
 
+            // Load the tax class from the product.  For configurable products, check the child first
             $product = $this->productRepository->getById($item->getProductId(), false, $order->getStoreId());
+            $children = $item->getChildrenItems();
 
-            if ($product->getTaxClassId()) {
+            if (!empty($children) && $children[0]->getProduct()->getTaxClassId()) {
+                $taxClass = $this->taxClassRepository->get($children[0]->getProduct()->getTaxClassId());
```

Comment:
@ryanreeves-taxjar Does this have any impact on how we'd handle child items for other types of products such as bundle products or just configurable? When calling `getChildrenItems`, do configurable products always return one simple child product? Should we add a check for configurable product type?

Reply:
Because of the exclusion checks that take place before this code (skipping certain items), there are no conflicts here.  I don't see any problems adding a check for configurable products, but it's not necessary.